src/app/profile/settings/notifications/toggle-bar.component.less

```diff
@@ -0,0 +1,66 @@
+@import (reference) '../../../../assets/stylesheets/shared/osio.less';
+
+.toggle {
+  display: none;
+  // add default box-sizing for this scope
+  &,
+  &:after,
+  &:before,
+  & *,
+  & *:after,
+  & *:before,
+  & + .toggle-btn {
+    box-sizing: border-box;
+    &::selection {
+      background: none;
+    }
+  }
+  + .toggle-btn {
+    outline: 0;
+    display: block;
+    width: 4em;
+    height: 2em;
+    position: relative;
+    cursor: pointer;
+    user-select: none;
+    &:after,
+    &:before {
+      position: relative;
+      display: block;
+      content: '';
+      width: 50%;
```

Comment:
You have percentages for width and height here, but are using `em` above - can you have these the same value type?